internal/agentlint/agentlint_test.go

```
@@ -0,0 +1,382 @@
+			}
+		}
+	}
+	return cmds
+}
+
+// runHelp runs "go run ./cmd/belayer [args...] --help" and returns stdout+stderr.
+func runHelp(t *testing.T, root string, args ...string) string {
+	t.Helper()
+	cmdArgs := append([]string{"run", "./cmd/belayer"}, args...)
+	cmdArgs = append(cmdArgs, "--help")
+	cmd := exec.Command("go", cmdArgs...)
```

golangci-lint (2.11.4)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)